apps/services/user-notification/src/app/modules/notifications/actor-notification.model.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  Table,
+  Column,
+  Model,
+  DataType,
+  AutoIncrement,
+  PrimaryKey,
+  CreatedAt,
+  UpdatedAt,
+  ForeignKey,
+  BelongsTo,
+} from 'sequelize-typescript'
+import {
+  CreationOptional,
+  InferAttributes,
+  InferCreationAttributes,
+} from 'sequelize'
+import { Notification } from './notification.model'
+
+@Table({
+  tableName: 'actor_notification',
+  indexes: [
+    { fields: ['user_notification_id'] },
+    { fields: ['message_id'] },
+    { fields: ['recipient'] },
+    { fields: ['on_behalf_of_national_id'] },
```

Comment:
Do you think there is a use case for the `message_id` index? Or `on_behalf_of_national_id` index if we join it from `userNotification.recipient`?

When querying actor notifications by recipient (for the admin), it might be good for the index to include `id`, `created` or whatever the query is sorted by. In case actors end up receiving A LOT of notifications.